source/mainnet/net/concepts/concepts-baker.rst

```diff
@@ -76,19 +78,49 @@ Finalization
+Finalization is the process by which a block is marked to be “finalized”, i.e., part of the authoritative chain. Transactions that are part of finalized blocks are considered authoritative. New blocks can be only added following the last finalized block to ensure the integrity of the chain. The finalization process is conducted periodically by the bakers with an effective stake amount of at least 0.1% of the total amount (effective stake) of CCD in pools, known as the Finalization committee. Total stake in pools is referred to as total stake in pools without Passive Delegation.
+
+Finalizers sign a block, and their collective signatures are aggregated to form a :ref:`quorum certificate<glossary-quorum-certificate>`. This quorum certificate is then included in the next block. When two blocks that are parent-child are in consecutive rounds in the same epoch and both have a quorum certificate, then the block in the first of these rounds (together with its ancestors) is considered finalized. Why isn't the child block considered to be final if it has a QC? This is to cover edge cases where network delays cause the QC of a block to not be received by the next block producer before a timeout. In that case, the block gets skipped by the next block producer and it cannot be considered final. To resolve this, only  the first among two consecutive certified blocks is considered to be final.
+
+Finalization happens at a minimum one second after block creation. A new block has to be created descended from that block for finalization to happen.
+
 When a sufficiently large number of members of the committee have received the block and agree on its outcome, the block is finalized. Newer blocks must have the finalized block as an ancestor to ensure the integrity of the chain.
 
 Finalization committee
 ----------------------
 
-The finalization committee is formed by the bakers with a staked amount of at least 0.1% of the total amount of existing CCD. This specifically implies that in order to participate in the finalization committee you will probably have to modify the staked amount to reach the threshold.
+The finalization committee is formed by the bakers with a staked amount of at least 0.1% of the total stake of CCD in pools. This specifically implies that in order to participate in the finalization committee you will probably have to modify the staked amount to reach the threshold. There is a minimum and maximum size for the finalization committee; it is 0.1% of the stake in pools, but always at least x bakers and always at most y bakers (e.g., x = 25 and y = 500).
```

Comment:
Why will you probably have to modify the staked amount? That's only the case for "small" bakers.